codec/console_reader.go

```diff
@@ -908,8 +911,13 @@ func (ctx *parseCtx) readInit(line string) error {
 	nodeVariant := chunks[1]
 	nodeVersion := chunks[2]
 
-	if !strings.HasPrefix(dmVersion, "2.") {
-		return fmt.Errorf("deepmind major version is unsupported (expected: 2.x, found %s)", dmVersion)
+	switch dmVersion {
+	case "2.0":
+		BlockVersion = 2
+	case "2.1":
+		BlockVersion = 3
+	default:
+		return fmt.Errorf("deepmind major version is unsupported (expected: 2.0 or 2.1, found %s)", dmVersion)
```

Comment:
Remove mention of `deepmind` to be replace by firehose

Reply:
I think we should beef up the message also, to point more clearly that `Firehose instrumented Geth` is incompatible. 
```suggestion
		return fmt.Errorf("major version of Firehose exchange protocol is unsupported (expected: 2.0 or 2.1, found %s), you are most probably running an incompatible version of the Firehose instrumented 'geth' client", dmVersion)
```